source/documentation/at-rules/use.html.md.erb

```diff
@@ -1,9 +1,431 @@
+  @mixin rounded {
+    border-radius: $radius;
+  }
+  ---
+  // style.scss
+  @use "src/corners" as *;
+
+  .button {
+    @include rounded;
+    padding: 5px + $radius;
+  }
+  ===
+  // src/_corners.sass
+  $radius: 5px
+
+  @mixin rounded
+    border-radius: $radius
+  ---
+  // style.sass
+  @use "src/corners" as *
+
+  .button
+    @include rounded
+    padding: 5px + $radius
+  ===
+  .button {
+    border-radius: 3px;
+    padding: 8px;
+  }
+<% end %>
+
+### Private Members
+
+As a stylesheet author, you may not want all the members you define to be
+available outside your stylesheet. Sass makes it easy to define a private member
+by starting its name with either `-` or `_`. These members will work just like
+normal within the stylesheet that defines them, but they won't be part of a
+module's public API. That means stylesheets that load your module can't see
+them!
+
+<% fun_fact do %>
+  If you want to make a member private to an entire *package* rather than just a
+  single module, just don't [forward its module][] from any of your package's
+  entrypoints (the stylesheets you tell your users to load to use your package).
+  You can even [hide that member][] while forwarding the rest of its module!
+
+  [forward its module]: forward
+  [hide that member]: forward#controlling-visibility
+<% end %>
+
+<% example(autogen_css: false) do %>
+  // src/_corners.scss
+  $-radius: 3px;
+
+  @mixin rounded {
+    border-radius: $-radius;
+  }
+  ---
+  // style.scss
+  @use "src/corners";
+
+  .button {
+    @include corners.rounded;
+
+    // This is an error! $-radius isn't visible outside of `_corners.scss`.
+    padding: 5px + corners.$-radius;
+  }
+  ===
+  // src/_corners.sass
+  $-radius: 5px
+
+  @mixin rounded
+    border-radius: $radius
+  ---
+  // style.sass
+  @use "src/corners"
+
+  .button
+    @include corners.rounded
+
+    // This is an error! $-radius isn't visible outside of `_corners.scss`.
+    padding: 5px + corners.$-radius
+<% end %>
+
+## Configuring Modules
+
+A stylesheet can define variables with the [`!default` flag][] to make them
+configurable. To load a module with configuration, write `@use <url> with
+(<variable>: <value>, <variable>: <value>)`. The configured values will override
+the variables' default values.
+
+[`!default` flag]: ../variables#default-values
+
+<%= partial '../../code-snippets/example-use-with' %>
+
+## Finding the Module
+
+It wouldn't be any fun to write out absolute URLs for every stylesheet you load,
+so Sass's algorithm for finding a module makes it a little easier. For starters,
+you don't have to explicitly write out the extension of the file you want to
+load; `@use "variables"` will automatically load `variables.scss`,
+`variables.sass`, or `variables.css`.
+
+<% heads_up do %>
+  To ensure that stylesheets work on every operating system, Sass loads files by
+  *URL*, not by *file path*. This means you need to use forward slashes, not
+  backslashes, even when you're on Windows.
```

Comment:
"even when you're on Windows" -> "even on Windows"

Reply:
Done.